### ISSUE TITLE
Get-LibreOffice fails on PowerShell Core

### ISSUE THREAD

**Issue by aaronparker** [aaronparker/Evergreen#5]
**Describe the bug**

`Get-LibreOffice` returns:

```
Exception calling ".ctor" with "1" argument(s): "Version string portion was too short or too long.                                                                  
Parameter name: input"                                                                                                                                              
At /Users/aaron/Projects/Evergreen/Evergreen/Public/Get-LibreOffice.ps1:56 char:41                                                                                  
+ ...      $version = [version]::new($($verblock | Select-Object -First 1))                                                                                         
+                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
+ FullyQualifiedErrorId : ArgumentException
```

**Comment by BronsonMagnan** [comment]
Aaron, I tested with Master branch, not seeing it, getting consistent URL output.

**Comment by aaronparker** [comment]
Works on Windows PowerShell but fails on PowerShell Core

**Comment by aaronparker** [comment]
Fixed in commit: 4c45344384a6a502e19aecbec8702ddb35a462c7
